source/common/tracing/opentracing_driver_impl.cc

```diff
@@ -164,8 +170,13 @@ SpanPtr OpenTracingDriver::startSpan(const Config&, Http::HeaderMap& request_hea
       tracerStats().span_context_extraction_error_.inc();
     }
   }
-  active_span = tracer.StartSpan(operation_name, {opentracing::ChildOf(parent_span_ctx.get()),
-                                                  opentracing::StartTimestamp(start_time)});
+  opentracing::StartSpanOptions options;
+  opentracing::ChildOf(parent_span_ctx.get()).Apply(options);
+  opentracing::StartTimestamp(start_time).Apply(options);
```

Comment:
This will work; though it would be more direct to write
```cpp
options.references.emplace_back(opentracing::SpanReferenceType::ChildOf, parent_span_context.get());
options.start_system_timestamp = start_time;
```
Those objects are really just to make specifying objects with the StartSpan method convenient.

Reply:
Had trouble making this change - compilation errors. Possibly this could be done as part of your PR?

Reply:
I made a typo (corrected now)
```cpp
options.references.emplace_back(opentracing::SpanReferenceType::ChildOf, parent_span_context.get());
```
should have been
```cpp
options.references.emplace_back(opentracing::SpanReferenceType::ChildOfRef, parent_span_context.get());
```
I can verify the code tomorrow. It shouldn't be dependent on any PR.